scripts/goldpolish-target.py

```diff
@@ -142,11 +143,15 @@ def main():
         args.ntLink,
     )
 
+    mapper=mapping_info[0]
```

Comment:
If mapping_info is always 
```suggestion
    mapper, s, x = mapping_info[:3]
```
To unpack altogether in one line. Or, can do 
```
mapper, s, x = mapping_info
```
If mapping_info is always of length 3